securitycenter/findingsv2/findings_test.go

```diff
@@ -0,0 +1,423 @@
+	if err := setupEntities(); err != nil {
+		fmt.Fprintf(os.Stderr, "Unable to initialize findings test environment: %v\n", err)
+		return
+	}
+	code := m.Run()
+	if err := cleanupEntities(); err != nil {
+		fmt.Fprintf(os.Stderr, "Unable to clean up findings test environment: %v\n", err)
+	}
+	os.Exit(code)
+}
+
+func TestCreateSource(t *testing.T) {
+	testutil.Retry(t, 5, 5*time.Second, func(r *testutil.R) {
+		orgID := setup(t)
+		buf := new(bytes.Buffer)
+
+		err := createSource(buf, orgID)
+
+		if err != nil {
+			r.Errorf("createSource(%s) had error: %v", orgID, err)
+			return
+		}
+
+		got := buf.String()
+		if want := "New source created"; !strings.Contains(got, want) {
+			r.Errorf("createSource(%s) got: %s want %s", orgID, got, want)
+		}
+		if !strings.Contains(got, orgID) {
+			r.Errorf("createSource(%s) got: %s want %s", orgID, got, orgID)
+		}
+	})
+}
+
+func TestGetSource(t *testing.T) {
+	setup(t)
+	testutil.Retry(t, 5, 5*time.Second, func(r *testutil.R) {
+		buf := new(bytes.Buffer)
+
+		err := getSource(buf, sourceName)
+
+		if err != nil {
+			r.Errorf("getSource(%s) had error: %v", sourceName, err)
+			return
+		}
+
+		got := buf.String()
+		if want := "Description: A new custom source that does X"; !strings.Contains(got, want) {
+			r.Errorf("getSource(%s) got: %s want %s", sourceName, got, want)
+		}
+		if !strings.Contains(got, orgID) {
+			r.Errorf("getSource(%s) got: %s want %s", sourceName, got, sourceName)
+		}
+	})
+}
+
+func TestListSources(t *testing.T) {
+	setup(t)
+	testutil.Retry(t, 5, 5*time.Second, func(r *testutil.R) {
+		buf := new(bytes.Buffer)
+
+		err := listSources(buf, orgID)
+
+		if err != nil {
+			r.Errorf("listSource(%s) had error: %v", orgID, err)
+			return
+		}
+
+		got := buf.String()
+		if want := "Description: A new custom source that does X"; !strings.Contains(got, want) {
+			r.Errorf("listSource(%s) got: %s want %s", orgID, got, want)
+		}
+		if !strings.Contains(got, sourceName) {
+			r.Errorf("listSource(%s) got: %s want %s", orgID, got, sourceName)
+		}
+	})
+}
+
+func TestUpdateSource(t *testing.T) {
+	setup(t)
+	testutil.Retry(t, 5, 5*time.Second, func(r *testutil.R) {
+		buf := new(bytes.Buffer)
+
+		err := updateSource(buf, sourceName)
+
+		if err != nil {
+			r.Errorf("updateSource(%s) had error: %v", sourceName, err)
+			return
+		}
+
+		got := buf.String()
+		if want := "Display name: New Display Name"; !strings.Contains(got, want) {
+			r.Errorf("updateSource(%s) got: %s want %s", sourceName, got, want)
+		}
+		if !strings.Contains(got, sourceName) {
+			r.Errorf("updateSource(%s) got: %s want %s", sourceName, got, sourceName)
+		}
+		if want := "does X"; !strings.Contains(got, want) {
+			r.Errorf("updateSource(%s) got: %s want %s", sourceName, got, want)
+		}
+
+	})
+}
+
+func TestListAllFindings(t *testing.T) {
+	// issue #3260 tracks this test skip.ß
+	t.Skip()
+	testutil.Retry(t, 5, 20*time.Second, func(r *testutil.R) {
+		orgID := setup(t)
+		buf := new(bytes.Buffer)
```

Comment:
Hi @vijaykanthm, in an effort to be more idiomatic and consistent with other samples, please refactor to `var buf bytes.Buffer` (and update usages appropriately, like line 273). Rather than comment on all the instances, please just search and replace all of them in the PR. Thanks!

Reply:
Thank you, addressed.